### PR TITLE
Conduit Documentation

### PR DESCRIPTION
**Pull request by winder**
## Summary

Took a stab at updating the conduit documentation. Some parts are still a little sparse, but I think there is a bit of coverage for everything.

## Test Plan

NA

### CODE REVIEW

File: docs/conduit/Development.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Creating A Plugin
```
Comment (AlgoStephenAkiki) [inline docs/conduit/Development.md:1]: Do we want to talk about what the "lifecycle" of a plugin is and where each function is expected to call?

File: docs/conduit/plugins/filter_processor.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## any / all
+One or more top-level operations should be provided. 
+* any: transactions are included if they match `any` of the nested sub expressions.
+* all: transactions are included if they match `all` of the nested sub expressions.
+
+If `any` and `all` are both provided, the transaction must pass both checks.
+
+## Sub expressions
+
+The sub expressions define a `tag` representing a field in the transaction. The full path to a given field should be provided, for example:
```
[the remark this answers is not in the document]
Comment (winder) [reply docs/conduit/plugins/filter_processor.md]: Thanks, I forgot to add those after looking up the dev portal links

File: docs/conduit/Development.md (hunk 1 above)
Comment (winder) [reply docs/conduit/Development.md:1]: Updated. This page probably should be updated to have a separate section for each interface. For now I think the basics are covered.

File: docs/conduit/plugins/home.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Plugin Configuration
+
+Each plugin is identified by a `Name`, and provided the `Config` during initialization.
+
+## Importers
+
+* [algod.md](algod.md)
```
Comment (Eric-Warehime) [inline docs/conduit/plugins/home.md:7]:
```suggestion
* [algod](algod.md)
```

**Review by Eric-Warehime — APPROVED** [review]

**Pull request merged**